nsxt/resource_nsxt_policy_transit_gateway_attachment_test.go

```diff
@@ -0,0 +1,178 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyTransitGatewayAttachmentCreateAttributes = map[string]string{
+	"display_name":    getAccTestResourceName(),
+	"description":     "terraform created",
+	"connection_path": "test-create",
```

Comment:
I'm a bit confused here - does this `connection_path` pass tests?  shouldn't this be policy path?

Reply:
OK we ignore this test as the vpc20 job only runs `^TestAcc.*V[Pp][Cc].*$` matching tests